target-platforms/packages/portal-master/package.json

```diff
@@ -0,0 +1,163 @@
+		"clay-card-grid": "2.22.4",
+		"clay-charts": "2.21.5",
+		"clay-checkbox": "2.22.4",
+		"clay-collapse": "2.22.4",
+		"clay-component": "2.22.4",
+		"clay-data-provider": "2.22.4",
+		"clay-dataset-display": "2.22.4",
+		"clay-dropdown": "2.22.4",
+		"clay-icon": "2.22.4",
+		"clay-label": "2.22.4",
+		"clay-link": "2.22.4",
+		"clay-list": "2.22.4",
+		"clay-loading-indicator": "2.22.4",
+		"clay-management-toolbar": "2.22.4",
+		"clay-modal": "2.22.4",
+		"clay-multi-select": "2.22.4",
+		"clay-navigation-bar": "2.22.4",
+		"clay-pagination": "2.22.4",
+		"clay-pagination-bar": "2.22.4",
+		"clay-portal": "2.22.4",
+		"clay-progress-bar": "2.22.4",
+		"clay-radio": "2.22.4",
+		"clay-select": "2.22.4",
+		"clay-sticker": "2.22.4",
+		"clay-table": "2.22.4",
+		"clay-tooltip": "2.22.4",
+		"d3": "5.9.3",
+		"d3-array": "1.2.4",
+		"d3-axis": "1.0.12",
+		"d3-brush": "1.0.6",
+		"d3-chord": "1.0.6",
+		"d3-collection": "1.0.7",
+		"d3-color": "1.2.3",
+		"d3-contour": "1.3.2",
+		"d3-dispatch": "1.0.5",
+		"d3-drag": "1.2.3",
+		"d3-dsv": "1.0.10",
+		"d3-ease": "1.0.5",
+		"d3-fetch": "1.1.2",
+		"d3-force": "1.2.0",
+		"d3-format": "1.3.2",
+		"d3-geo": "1.11.3",
+		"d3-hierarchy": "1.1.8",
+		"d3-interpolate": "1.4.0",
+		"d3-path": "1.0.7",
+		"d3-polygon": "1.0.5",
+		"d3-quadtree": "1.0.5",
+		"d3-random": "1.1.2",
+		"d3-scale": "2.2.2",
+		"d3-scale-chromatic": "1.3.3",
+		"d3-selection": "1.4.0",
+		"d3-shape": "1.3.7",
+		"d3-time": "1.0.10",
+		"d3-time-format": "2.1.3",
+		"d3-timer": "1.0.9",
+		"d3-transition": "1.2.0",
+		"d3-voronoi": "1.1.4",
+		"d3-zoom": "1.7.3",
+		"formik": "2.1.4",
+		"incremental-dom": "0.5.1",
+		"incremental-dom-string": "0.0.2",
+		"metal": "2.16.8",
+		"metal-affix": "2.0.0",
+		"metal-ajax": "2.1.1",
+		"metal-anim": "2.0.1",
+		"metal-aop": "3.0.0",
+		"metal-assertions": "2.16.8",
+		"metal-clipboard": "2.0.1",
+		"metal-component": "2.16.8",
+		"metal-debounce": "2.0.2",
+		"metal-dom": "2.16.8",
+		"metal-drag-drop": "3.3.1",
+		"metal-events": "2.16.8",
+		"metal-incremental-dom": "2.16.9",
+		"metal-jsx": "2.16.8",
+		"metal-key": "1.0.0",
+		"metal-keyboard-focus": "2.0.1",
+		"metal-multimap": "2.0.0",
+		"metal-pagination": "2.0.1",
+		"metal-path-parser": "1.0.4",
+		"metal-position": "2.1.2",
+		"metal-promise": "3.0.5",
+		"metal-router": "3.6.3",
+		"metal-scrollspy": "2.1.1",
+		"metal-soy": "2.16.9",
+		"metal-soy-bundle": "2.16.9",
+		"metal-state": "2.16.8",
+		"metal-storage": "2.0.1",
+		"metal-structs": "1.0.2",
+		"metal-throttle": "3.0.1",
+		"metal-toggler": "2.3.0",
+		"metal-uri": "2.4.0",
+		"metal-useragent": "3.0.1",
+		"metal-web-component": "2.16.8",
+		"prop-types": "15.7.2",
+		"querystring": "0.2.0",
+		"react": "16.12.0",
+		"react-dnd": "11.1.1",
+		"react-dnd-html5-backend": "11.1.1",
+		"react-dom": "16.12.0",
+		"recharts": "1.8.5",
+		"senna": "2.7.9",
+		"xss-filters": "1.2.7"
+	},
+	"description": "Target Platform for liferay-portal-master",
```

Comment:
Can we also add a script to this package.json for `prepublishOnly ` and add a warning to prevent publishing to npm?

Reply:
Yeah, makes sense :+1:

Reply:
Done